project_PassMeCash_Polkadot.md

```diff
@@ -0,0 +1,165 @@
+4. PassMeCash Flow link https://miro.com/app/board/uXjVPqS1sPY=/
+5. PassMeCash Documentation link: https://docs.google.com/document/d/1tgbiOFsb4JE6mHmP2oG-XP5kriGgPhun118o5174e4w/edit?usp=sharing
+6. Github Repo: https://github.com/passmecash
+
+
+The technology stacks to be used include:
+- Figma
+- Miro
+- Tailwind
+- React.js
+- Javascript
+- Python
+- Django
+- Ink
+- Chainlink aggregator
+- Web3.py
+
+
+### Ecosystem Fit
+
+The **PassMeCash** is built for all users of the ecosystem and it can be leveraged upon to send and receive payments globally. Users can also use crypto (DOT) to make transaction payments for goods and services via the PassMeCash platform. This will lead to more adoptability and usability of Polkadot.
+
+
+## Team :busts_in_silhouette:
+
+### Team members
+Team lead is Onose Oghenevwogaga. He is a blockchain community advocate from Nigeria.
+
+### Contact
+
+- **Contact Name:** Onose Oghenevwogaga
+- **Contact Email:** onoseogaga@gmail.com
+
+
+### Legal Structure
+
+- **Registered Address:** Majemite Compound, #28 Jakpa Road, Effurun, Delta state, Nigeria.
+- **Registered Legal Entity:** CodeUp Technologies Limited
+
+### Team's experience
+
+CodeUp Technologies Team has competent UI/UX desiner, Frontend & Backend developers as well as Smart Contract Developers that is experienced with Polkadot.
+Thus there is enough requisite experience by the team to deliver this project as proposed.
+ 
+
+### Team Code Repos
+
+- https://github.com/passmecash
+
+Please also provide the GitHub accounts of all team members. If they contain no activity, references to projects hosted elsewhere or live are also fine.
+ 
+- https://github.com/iamgaddiel
+- https://github.com/favourwilson
+- https://github.com/scofieldwhitefire
+- https://github.com/taofeek-ob
+- https://github.com/frankudoags
+- https://github.com/emmanuel0705
+- https://github.com/Dani-ui
+- https://www.behance.net/TochuCU
+- https://www.behance.net/clementfadeyi
+- https://github.com/ogasky
+
+### Team LinkedIn Profiles (if available)
+
+- https://www.linkedin.com/in/onose-ogaga
+
+
+## Development Status :open_book:
+
+Details of the development of PassMeCash have been stated above with relevant links and we are working towards achieving our first MVP for this project, however we are to add some pages in the designs base on the recent flow diagram from the Team.
+
+
+## Development Roadmap :nut_and_bolt:
+
+This section should break the development roadmap down into milestones and deliverables. To assist you in defining it, we have created a document with examples for some grant categories [here](../docs/Support%20Docs/grant_guidelines_per_category.md). Since these will be part of the agreement, it helps to describe _the functionality we should expect in as much detail as possible_, plus how we can verify and test that functionality. Whenever milestones are delivered, we refer to this document to ensure that everything has been delivered as expected.
```

Comment:
There is still a lot of text included from the default template. Can you remove these parts and replace with your own?

Reply:
Done.